prime-router/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml

```diff
@@ -13,7 +13,10 @@ elements:
     schema: ../common/message-header
 
   - name: software-segment
-    condition: 'Bundle.entry.resource.ofType(MessageHeader).exists()'
+    condition: >
+      Bundle.entry.resource.ofType(MessageHeader).exists() and 
+      Bundle.entry.resource.ofType(MessageHeader).source.exists() and
+      Bundle.entry.resource.ofType(MessageHeader).source.extension(%`rsext-software-vendor-org`).exists()
```

Comment:
This isn't the only place this applies, but I'm curious: If the sender is sending us FHIR and they do not include this custom RS extension, we will not map their message properly and the ONLY way to tell this is to manually compare the FHIR to the HL7? That means onboarding of FHIR senders will be even more tedious than onboarding HL7v2 and probably cannot be done in an automated fashion (or with the help of self-service tools)?

Reply:
I think onboarding any sender/receiver pair where the input format != the output format will be a little tedious.  I do think for this case of FHIR -> HL7 the solution is to produce the list of extensions that must be populated in order to reliably translate a FHIR message to HL7.

I'm not sure there is a way around this and explicitly always create HL7 from FHIR

Reply:
Yep. going to revert this change and move it into the test schema.  This will be covered by the mapping MSH/SFT->MessageHeader->MSH/SFT and I'm not convinced that this is the only extension we need to look for when deciding whether to create an SFT segment